desktop/start.js

```diff
@@ -8,7 +8,7 @@ const basePort = 8080;
 portfinder.getPortPromise({
     port: basePort,
 }).then((port) => {
-    const devServer = `webpack-dev-server --config config/webpack/webpack.dev.js --port ${port} --env.platform desktop`;
+    const devServer = `webpack-dev-server --config config/webpack/webpack.dev.js --port ${port} --env platform=desktop`;
```

Comment:
Is this change necessary?

Reply:
In Webpack v5, syntax on env variables has changed to be `--env  key=value`


Reference: https://webpack.js.org/guides/environment-variables/#root